Gems/Multiplayer/Code/Source/MultiplayerSystemComponent.cpp

```diff
@@ -134,59 +184,59 @@ namespace Multiplayer
                 ->Attribute(AZ::Script::Attributes::Module, "multiplayer")
                 ->Attribute(AZ::Script::Attributes::Category, "Multiplayer")
                 ->Method("GetOnEndpointDisconnectedEvent", [](AZ::EntityId id) -> EndpointDisconnectedEvent*
-                {
```

Comment:
Did you touch anything here? If you didn't, can you revert all these whitespace changes so it's easier to actually see what was touched?

Reply:
Reverted whitespace changes. It should be good now.